documentation/overview.md

```diff
@@ -32,14 +32,15 @@ Datastores is an abstraction which hides storage details and provide uniform int
 * VVol(Virtual Volumes) - Virtual Volumes datastore represents a storage container in vCenter Server and vSphere Web Client. 
 
 ## Kubernetes Storage support
-vSphere Cloud Provider supports every storage primitive exposed by Kubernetes -
-* Volumes
-* Persistent Volumes (PV)
-* Persistent Volumes Claims (PVC)
-* Storage Class
-* Stateful Sets
-
-Persistent volumes requested by stateful containerized applications can be provisioned on VSAN, VVol, VMFS or NFS datastores.
+vSphere Cloud Provider (VCP) supports every storage primitive exposed by Kubernetes -
```

Comment:
Shouldn't we use vSphere storage for Kubernetes here?

Reply:
+1

Reply:
No, Above in the documentation we have mentioned
> vSphere Storage for Kubernetes offers a Cloud Provider (vSphere Cloud Provider - VCP) for Kubernetes which allows Pods to use enterprise grade persistent storage.

This sets the context for VCP. Under `vSphere Storage for Kubernetes` we will have multiple offerings. VCP is just one of them.